guitar-api.yaml

```diff
@@ -27,8 +27,8 @@ paths:
       description: returns a single guitar
       operationId: getGuitarByID
       parameters:
-        - name: guitarId
-          in: query
+        - name: guitar_id
+          in: path
```

Comment:
Note: when you have a `path` param, the URL must contain that path param. So now you will have both `/guitars` and `/guitars/{guitar_id}`

(also, as a convention, **plural is preferred over singular.** with some minor exceptions)